### PR TITLE
Please review this, @casey-collab

### PR DESCRIPTION
**Pull request by pbyrne98**
Hi, not sure how the newline issue got in there. It should have been a simple bot add.

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this.

File: english/lasso-hand/ride/spanking.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+You decide you've been bad and deserve a spanking.
```
Comment (casey-collab) [inline english/lasso-hand/ride/spanking.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@pbyrne98, that happens because there was no newline at the end of the file, so by adding to the file, you had to add a newline to that line. This comes up often enough that I helped Casey understand how to deal with it.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**